openedx/core/djangoapps/user_authn/views/tests/test_password.py

```diff
@@ -176,6 +177,36 @@ def test_password_change(self):
         response_dict = json.loads(response.content.decode('utf-8'))
         assert response_dict['success']
 
+    @ddt.data(
+        (True, True, 'walter@graymattertech.com'),
+        (True, False, 'walter@graymattertech.com'),
+        (False, True, 'walter@graymattertech.com'),
```

Comment:
you can reference OLD_EMAIL here instead of writing the explicit string

Reply:
Ah, I tried using that with self. I'm adding this rn.